README.md

```diff
@@ -103,31 +103,37 @@ Keep-Alive: timeout=5
+| cors.allowedHeaders       | `CORS_ALLOWED_HEADERS`           | n/a             |    no    | Headers to allow for CORS                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| cors.exposedHeaders       | `CORS_EXPOSED_HEADERS`           | "ETag"          |    no    | Headers to expose for CORS                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| cors.credentials          | `CORS_CREDENTIALS`               | `false`         |    no    | Allow credentials in CORS requests                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| cors.preflightContinue    | `CORS_PREFLIGHT_CONTINUE`        | `false`         |    no    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| cors.optionsSuccessStatus | `CORS_OPTIONS_SUCCESS_STATUS`    | 204             |    no    | If the Options call passes CORS, which http status to return                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Super! Just one small request: could you order these alphabetically together with the rest of the "cors.<propertyName>" options?

Reply:
Done in 966ca98